demo/mobile/src/app/app.js

```diff
@@ -22,6 +25,112 @@ angular.module('wfm-mobile', [
   }),
   require('@raincatcher/vehicle-inspection')
 ]);
+
+var auth = {};
+
+// initialise the keycloak js adapter if keycloak configuration has been specified
+if (keycloakConfig) {
+  // require keycloak js client adapter
+  var Keycloak = require('keycloak-js');
+
+  // keycloak init config
+  var initConfig = {onLoad: 'login-required'};
+
+  // the keycloak json config
+  var keycloakJS = Keycloak(keycloakConfig);
+
+  angular.element(document).ready(function() {
+    // initialise the Keycloak JS Adapter
+    keycloakJS.init(initConfig).success(function() {
+      auth.keycloak = keycloakJS;
+
+      console.log("Keycloak Ininitalisation Success");
+
+      // load keycloak user profile
+      auth.keycloak.loadUserProfile().success(function(profile) {
+        auth.keycloak.userProfile = profile;
+      }).error(function(err) {
+        console.log("Failed to Load User Profile", err);
```

Comment:
We can use cordova based dialogs. I'm more interested in actual error handling rather than dialog itself. If error will happen we will miss user and literally there is no sense to have ui. 
We will need to go back to login screen. If we really want to showcase integration we should at least provide proper error handling.